.evergreen/.evg.yml

```diff
@@ -2216,6 +2339,27 @@ buildvariants:
   tasks:
     - name: "test_atlas_task_group_search_indexes"
 
+- name: "oidc-auth-test"
+  display_name: "OIDC Auth"
+  run_on: ubuntu2204-small
```

Comment:
We explicitly test on Windows and MacOS in PyMongo, but I'd consider that optional.

Reply:
All of our tests use linux/ubuntu as the `os`, so I took those out.